src/Symfony/Component/Translation/Translator.php

```diff
@@ -174,6 +210,13 @@ public function trans(?string $id, array $parameters = [], ?string $domain = nul
             }
         }
 
+        if ([] !== $this->globalTranslatableParameters && !isset($this->globalParameters[$locale])) {
```

Comment:
we don't use empty-values comparisons in Symfony (same everywhere and for your next PRs ;) )
```suggestion
        if ($this->globalTranslatableParameters && !isset($this->globalParameters[$locale])) {
```

Reply:
Noted, thanks :slightly_smiling_face: